extensions/amp-lightbox-viewer/0.1/service/lightbox-manager-impl.js

```diff
@@ -54,19 +53,11 @@ export class LightboxManager {
      **/
     this.initPromise_ = null;
 
```

Comment:
The intention of this is lazy initialization. Now `maybeInit_` which calls `scanLightboxables_` under the hood will get called at `getElements` but not before.